### PR TITLE
Recorded Future enrichment update

### PR DESCRIPTION
**Pull request by RecordedFutureOskbo**
Required items, please complete
   
   Change(s):
   - Added detections
   - Changed format of logotype from svg to png
   - Improved layout of enrichment comments
   - Updated readme

   Reason for Change(s):
   - New backend functionality where we surface more data.

   Testing Completed:
   - Yes

### CODE REVIEW

**Comment by v-sabiraj** [comment]
@RecordedFutureOskbo, Solutions/Recorded Future/Data/RecordedFutureLogo.png, please check the path for the logo it should be added in [logos folder](https://github.com/Azure/Azure-Sentinel/tree/master/Logos), thanks.

**Comment by v-sabiraj** [comment]
@manishkumar1991/ @rahul0216, can you please review the changes for playbooks, thanks.

File: Solutions/Recorded Future/Data/Solution_RecordedFuture.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "Workbooks/Recorded Future - Actively Communicating C&C IPs to DNS Events - Correlation&Threat Hunting.json"
   ],
   "BasePath": "C:\\GitHub\\Azure-Sentinel\\Solutions\\Recorded Future",
-  "Version": "2.2.2",
+  "Version": "2.3.0",
```
Comment (v-sabiraj) [inline Solutions/Recorded Future/Data/Solution_RecordedFuture.json:28]: @RecordedFutureOskbo, can you please check the version for solution, should it be 2.2.3?

**Comment by v-sabiraj** [comment]
@RecordedFutureOskbo, also please if you need to add the new logo in readme.md, thanks.
image

File: Solutions/Recorded Future/Data/Solution_RecordedFuture.json (hunk 1 above)
Comment (RecordedFutureOskbo) [reply Solutions/Recorded Future/Data/Solution_RecordedFuture.json:28]: We communicated this release as 2.3 internally due to the added detections.

**Review by rahul0216 — APPROVED** [review]
Updates look good.

**Pull request merged**